realm-transformer/src/main/groovy/io/realm/transformer/RealmTransformer.groovy

```diff
@@ -278,4 +288,23 @@ class RealmTransformer extends Transform {
         return merged;
     }
 
+    // Use reflection to get the AndroidBuild from Android plugin for getting the path to android.jar
+    // There is no official way to get the path to android.jar for transform.
+    // See https://android.googlesource.com/platform/tools/base/+/gradle_2.0.0/build-system/gradle-core/src/main/groovy/com/android/build/gradle/internal/transforms/InstantRunTransform.java#337
+    private void addBootClassesToClassPool(ClassPool classPool) {
+        try {
+            def androidBuilderField = BasePlugin.getDeclaredField("androidBuilder");
+            androidBuilderField.setAccessible(true)
+            AndroidBuilder androidBuilder = (AndroidBuilder)androidBuilderField.get(androidPlugin)
+            def bootClasspathList = androidBuilder.getBootClasspathAsStrings(true)
+            bootClasspathList.each {
+                logger.info "Add boot class " + it + " to class pool."
+                classPool.appendClassPath(it)
+            }
+        } catch (Exception e) {
+            // Just log it. It might not impact the transforming if the method which needs to be transformer doesn't
+            // contain classes from android.jar.
+            logger.info("Cannot get androidBuilder caused by:", e)
```

Comment:
I think this is good enough and saves us some time to transfer the issue from helpscout to github. People will create an issue here anyway.